tests/integ/test_auto_ml.py

```diff
@@ -142,7 +142,9 @@ def test_auto_ml_invalid_target_attribute(sagemaker_session):
     job_name = unique_name_from_base("auto-ml", max_length=32)
     inputs = sagemaker_session.upload_data(path=TRAINING_DATA, key_prefix=PREFIX + "/input")
     with pytest.raises(
-        UnexpectedStatusException, match="Could not complete the data builder processing job."
+        ClientError,
+        match="An error occurred (ValidationException) when calling the CreateAutoMLJob "
```

Comment:
These need to be escaped in a `match` expression in this case.
```suggestion
        match="An error occurred \(ValidationException\) when calling the CreateAutoMLJob "
```